qa/test_framework/test_framework.py

```diff
@@ -189,12 +192,18 @@ def main(self, options=["--disablewallet", "--testnet", "--disableexchangerates"
         parser.add_argument('-d', '--bitcoind', help="the bitcoind binary")
         parser.add_argument('-t', '--tempdir', action='store_true', help="temp directory to store the data folders", default="/tmp/")
         parser.add_argument('-c', '--cointype', help="cointype to test", default="BTC")
+        parser.add_argument('-T', '--Tor', help="use tor", default="no")
         args = parser.parse_args(sys.argv[1:])
         self.binary = args.binary
         self.temp_dir = args.tempdir
         self.bitcoind = args.bitcoind
         self.cointype = args.cointype
+        self.useTor = args.Tor
         self.options = options
+        if args.Tor == "yes":
```

Comment:
Not that big of a deal but this could be a boolean rather than a string from the command line.